### ISSUE TITLE
Can climate oscillation indices regions be added

### ISSUE THREAD

**Issue by ahuang11** [regionmask/regionmask#243]
e.g. Nino 1+2/3/4/3.4, MJO, NAO, etc

**Comment by mathause** [comment]
The NINO indices should be easy to add - these are just square boxes, right? As they overlap (at least NINO3.4) we'll have to do them individually. I am fine to add them but they should also be trivial to [create by hand](https://regionmask.readthedocs.io/en/stable/notebooks/create_own_regions.html) (or even just use `ds.sel()`).

Are there agreed upon definitions for MJO and NAO regions? How should we organize them? `regionmask.defined_regions.indices.NINO34`?

**Comment by ahuang11** [comment]
Correct, but I hate remembering/looking up what the lons/lats are exactly; would love to just mask by using label (thus the feature request).

I guess CPC for NAO/PNA/AAO/AO?
https://www.cpc.ncep.noaa.gov/products/precip/CWlink/pna/nao.shtml

And the classic Wheeler and Hendon paper for MJO:
https://journals.ametsoc.org/view/journals/mwre/132/8/1520-0493_2004_132_1917_aarmmi_2.0.co_2.xml

Maybe regionmask.defined_regions.climate_oscillations.nino34?
